### ISSUE TITLE
ValueError: signal only works in main thread of the main interpreter

### ISSUE THREAD

**Issue by ZenoNing** [maria-korosteleva/GarmentCode#15]
Hi Maria, first I want to express my gratitude for your brilliant work! It is immensely useful for my target.

When I ran `python gui.py`, the GarmentCode GUI was successfully displayed. However, when I selected the meta pattern and tried to do some draping for the current design, some errors happened.

```
Progress: [--------------------------------------------------] 0.0%Sim::Configured_design::crashed with signal only works in main thread of the main interpreter
Traceback (most recent call last):
  File "/home/billyhe/zenoning/GarmentCode/pygarment/meshgen/simulation.py", line 178, in run_sim
    sim_frame_sequence(garment, config, store_usd, verbose=verbose)
  File "/home/billyhe/zenoning/GarmentCode/pygarment/meshgen/simulation.py", line 128, in sim_frame_sequence
    signal.signal(signal.SIGALRM, alarm_handler)
  File "/home/billyhe/anaconda3/envs/garmentcode/lib/python3.9/signal.py", line 56, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
```

I checked the code and commented out the 128 line: `signal.signal(signal.SIGALRM, alarm_handler)` in simulation.py, it worked. 

Based on the above reasons, I want to get some advice from you, could you please help me out?

**Comment by maria-korosteleva** [comment]
Oh! That's a great catch! 

I developed GUI on a Windows machine, and the `signal.signal()` is only used for Linux/MacOS, so I didn't notice an issue. The functionality is made to crash the simulation in case when it gets stuck, so it should not be removed entirely. I will take a closer look at how it should be properly done for GUI. 

Please, you are welcome to ask questions! If they are not related to the one you are sharing here, please, open a new GitHub issue for topic.

**Comment by maria-korosteleva** [comment]
Fixed in [release v2.0.1](https://github.com/maria-korosteleva/GarmentCode/releases/tag/v2.0.1). Update the `pygarment` library to [pygarment 2.0.1](https://pypi.org/project/pygarment/2.0.1/)
